astropy/io/misc/asdf/tags/transform/polynomial.py

```diff
@@ -274,15 +278,16 @@ def to_tree_transform(cls, model, ctx):
                     coefficients[i, j] = getattr(model, name).value
         node = {'polynomial_type': poly_type, 'coefficients': coefficients}
         if ndim == 1:
-            if model.domain is not None:
+            if model.domain != model._default_domain_window['domain']:
                 node['domain'] = model.domain
-            if model.window is not None:
+            if model.window != model._default_domain_window['window']:
                 node['window'] = model.window
         else:
-            if model.x_domain or model.y_domain is not None:
-                node['domain'] = (model.x_domain, model.y_domain)
-            if model.x_window or model.y_window is not None:
-                node['window'] = (model.x_window, model.y_window)
+            for attr in ['x_domain', 'y_domain', 'x_window', 'y_window']:
+                attr_obj = getattr(model, attr)
+                if attr_obj != model._default_domain_window[attr]:
+                    node[attr] = attr_obj
+            
```

Comment:
Does the change in the serialization attributes need to be reflected in the schema? I forget what was put in previously as far as the schema goes (perhaps ignored).

Reply:
Also, did we release the previous serialization? If so, do we have a backwards compatibility issue?